components/table/table-wrapper.js

```diff
@@ -62,6 +63,16 @@ export const tableStyles = css`
 	th.d2l-table-header-col-sortable {
 		padding: 0;
 	}
+	.d2l-checkbox,
+	d2l-input-checkbox,
+	d2l-selection-select-all,
+	input[type="checkbox"] {
+		/* Get the margin required to make the height 48px based on the checkbox height */
+		margin-block: calc(0.5 * (48px - ${cssSizes.inputBoxSize}rem - 2 * var(--d2l-table-cell-padding)) - 1px);
```

Comment:
It would probably be helpful to use a css var for the `48px`. Then we can also use that when we update the heights of the regular cells.